retail/interactive-tutorial/TestResourcesSetupCleanup/README.md

```diff
@@ -0,0 +1,50 @@
+# How to set up/ tear down the test resources
+
+## Required environment variables
+
+To successfully import the catalog data for tests, the following environment variables should be set:
+ - GOOGLE_CLOUD_PROJECT_NUMBER
+ - GOOGLE_CLOUD_PROJECT_ID 
+ - BUCKET_NAME
+ - EVENTS_BUCKET_NAME
+
+These values are stored in the Secret Manager and will be submitted as 
+   docker environment variables before the test run.
+
+The Secret Manager name is set in .kokoro/presubmit/common.cfg file, SECRET_MANAGER_KEYS variable.
```

Comment:
This informatio is not relevant for users.
```suggestion
```

Reply:
done